components/infra-proxy-service/migrations/pipeline/phaseonemigration.go

```diff
@@ -119,7 +119,7 @@ func parseOrgUserAssociationSrc(result <-chan PipelineData, service *service.Ser
 		for res := range result {
 			_, err := service.Migration.StartUserAssociationParsing(res.Ctx, res.Result.Meta.MigrationID, res.Result.Meta.ServerID)
 			if err != nil {
-				log.Errorf("Failed to update the status for start org user association for the migration id %s : %s", res.Result.Meta.MigrationID, err.Error())
+				log.Errorf("Failed to update the status for start org user association parsing for the migration id %s : %s", res.Result.Meta.MigrationID, err.Error())
```

Comment:
I am concerned that in these cases, since we are not sending the response in the channel, the caller may wait for the data in the channel. Ideally we should be sending back the error too

Reply:
Returned an error in the channel. But I have repeated the code in case of error to add in the channel. Is it fine?
Tried to do it in defer but defer won't execute until the result channel is closed.

Reply:
It looks ok to me but please test it.

Reply:
It looks ok to me but please test it.

Reply:
sure will test it